examples/cpp/customserializer/main.cpp

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <iostream>
+
+#include <geode/CacheFactory.hpp>
+#include <geode/PoolManager.hpp>
+#include <geode/PdxSerializer.hpp>
+#include <geode/PdxWrapper.hpp>
+#include "Order.hpp"
+#include "OrderSerializer.hpp"
+
+using namespace apache::geode::client;
+using namespace customserializer;
+
+int main(int argc, char **argv) {
+  auto cacheFactory = CacheFactory();
+  cacheFactory.set("log-level", "none");
+  auto cache = cacheFactory.create();
+  auto poolFactory = cache.getPoolManager().createFactory();
+
+  poolFactory->addLocator("localhost", 10334);
+  auto pool = poolFactory->create("pool");
+  auto regionFactory = cache.createRegionFactory(RegionShortcut::PROXY);
+  auto region = regionFactory.setPoolName("pool").create("custom_orders");
+
+  auto orderSer = std::make_shared<OrderSerializer>();
+  cache.getTypeRegistry().registerPdxSerializer(orderSer);
```

Comment:
I don't think we actually need to register a serializer when we use the pdx wrapper model. If you do then it seems rather redundant to have to specify it on the `PdxWrapper`. We should test this and write up a ticket to address this if you must register the serializer and pass it to the wrapper.

Reply:
It is required, which is odd.  We should get a ticket for it.